internal/pkg/log/logger.go

```diff
@@ -52,6 +55,86 @@ func (l *zapLogger) Log(level string, args ...any) {
 	}
 }
 
+func getMessage(args []any) string {
+	if len(args) == 1 {
+		if str, ok := args[0].(string); ok {
+			return str
+		}
+	}
+	return fmt.Sprint(args...)
+}
+
+func (l *zapLogger) LogCtx(ctx context.Context, level string, args ...any) {
+	message := getMessage(args)
```

Comment:
Could we please just add the unused ctx argument to the beginning of the method in this step?

We have not yet agreed on what the final methods will look like, see our discussion on slack.



![image](https://github.com/keboola/keboola-as-code/assets/19371734/10da2768-596c-4866-865c-3ca51c7a4583)